app/moviesPage/index.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+function MainController($scope, $http) {
+
+    const key = 'fc298428bb77d2a10fb5e0bc411eb836';
+    const API = 'https://api.themoviedb.org/3/movie/now_playing?api_key=';
+    const movieDetails = "https://api.themoviedb.org/3/movie/";
```

Comment:
Check JS version and and see if we can use-interpolation
`https://api.themoviedb.org/3/movie/now_playing?api_key=${}`;